webgrid/tests/test_client_helpers.py

```diff
@@ -0,0 +1,160 @@
+from io import BytesIO
+
+import xlwt
+from nose.tools import assert_raises
+
+from . import client_helpers
+
+
+class Test_assert_equal:
```

Comment:
please follow PEP8 naming conventions.  Class name should be: `TestAssertEqual`

This also applies to the other test classes in this module.

Reply:
I typically name my tests 'Test_<thing_being_tested>', but I'll switch to PEP8 names for the tests.

This has been done.